### DISCUSSION TITLE
What is Media URL prefix to get the media received

### DISCUSSION THREAD

**Discussion by vsnshanmuganathan** [aldinokemal/go-whatsapp-web-multidevice#205] [Q&A] [answered]
In my webhook i get something like
```media path: statics/media/1728808709-14e942-5c00-483d-b3fd-68608ba1e90c.jpeg```
but how to download or view the image/media files?

**Comment by aldinokemal** [comment, marked as answer]
You can download by adding your app domain name, as it is stored locally.
